Final_Project_Paper-AR.qmd

```diff
@@ -0,0 +1,174 @@
+
+Over the last century, the Foreign Policy Association (FPA) has been producing content on U.S. foreign policy. Originally established as the League of Free Nations Association, the association’s initial aspirations were to support peace initiatives promulgated by Woodrow Wilson. However, the League of Free Nations Association was short-lived, and the association would shortly be changed to the FPA in 1923, where all matters of U.S. foreign policy fell under their purview of study. Moreover, the FPA had high-ranking U.S. political figures among its incorporators, including Eleanor Roosevelt and John Foster Dulles. In addition to these important incorporators, FPA members also had addresses delivered before them by Franklin D. Roosevelt.^[fn3: Foreign Policy Association, “About FPA,” accessed April 28, 2024, https://www.fpa.org/about/] In 1931, the FPA began producing the Foreign Policy Bulletin, which published articles on international events affecting U.S. foreign policy. The Foreign Policy Bulletin Dataset is comprised of the volumes produced by the Foreign Policy Bulletin between 1936 and 1946.^[fn4: John Mark Ockerbloom, “The Online Books Page: Foreign Policy Bulletin,” accessed April 28, 2024, https://onlinebooks.library.upenn.edu/webbin/serial?id=fpabulletin]
+
+```{r}
+#Load in Libraries
+library(tidytext)
+library(tidyverse)
+library(readtext)
+library(tm)
+library(topicmodels)
+library(widyr)
+library(SnowballC)
+library(DigitalMethodsData)
+library(wordVectors)
+library(magrittr)
+library(dplyr)
+library(ggplot2)
+```
+
+```{r}
+#load in dataset
+metadata <- read.csv("ForeignPolicyBulletinData/metadata.csv")
```

Comment:
You should add the github repository for ForeignPolicyBulletinData as a git submodule. Every time I clone your repo I can't run the code because the files from this rep are not included. See more about git submodules here: https://git-scm.com/book/en/v2/Git-Tools-Submodules